tests/TestHelpers/GraphHelper.php

```diff
@@ -1598,19 +1598,21 @@ public static function sendSharingInvitation(
 		string $password,
 		string $spaceId,
 		string $itemId,
-		string $shareeId,
-		string $shareType,
+		array $shareeIds,
+		array $shareTypes,
 		?string $permissionsRole,
 		?string $permissionsAction,
 		?string $expireDate
 	): ResponseInterface {
 		$url = self::getBetaFullUrl($baseUrl, "drives/$spaceId/items/$itemId/invite");
 		$body = [];
 
-		$recipients['objectId'] = $shareeId;
-		$recipients['@libre.graph.recipient.type'] = $shareType;
-
-		$body['recipients'] = [$recipients];
+		for ($i = 0; $i < \count($shareeIds); $i++) {
+			$body['recipients'][] = [
+				"@libre.graph.recipient.type" => $shareTypes[$i],
+				"objectId" => $shareeIds[$i]
+			];
+		}
```

Comment:
```suggestion
		foreach ($shareeIds as $index => $shareeId) {
		        $shareType = $shareTypes[$index];
			$body['recipients'][] = [
				"@libre.graph.recipient.type" => $shareType,
				"objectId" => $shareeId
			];
		}
```
Can we do in this way?